staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service.go

```diff
@@ -69,6 +69,7 @@ func NewGRPCService(endpoint string, callTimeout time.Duration) (Service, error)
 			if err != nil {
 				klog.Errorf("failed to create connection to unix socket: %s, error: %v", addr, err)
 			}
+			klog.V(3).Infof("Successfully dialed Unix socket %v", addr)
```

Comment:
NewGRPCService logs at v4, and the dialer can be called more often... is v4 more appropriate?

Reply:
Done.